_data/conf/main.yml

```diff
@@ -108,7 +108,7 @@ image_viewer_error_add_path: true
 #                Image lazy loader
 ###########################################################
 # this enables image lazy loader
-image_lazy_loader: true
+image_lazy_loader: false
```

Comment:
Would you mind to explain this change?

Reply:
This improves some of the performance to the web page.

Reply:
This comment should be in the code, so anybody can understand it while reading it.